src/mixins/_ms.scss

```diff
@@ -0,0 +1,7 @@
+@use "sass:math";
+@use "../design-tokens/modular-scale.yml";
+
+@function step($step, $base: 1em, $ratio: modular-scale.$ratio) {
```

Comment:
Nit: Should the base value be a design token as well?

Reply:
Why?

Reply:
I see this as more the base of the current value output. Most of the time you will want to output `em`, but sometimes you may want to output `rem`, etc.

Reply:
Oh yeah, that's clever :+1: